### DISCUSSION TITLE
Cross origin failed when using Axios

### DISCUSSION THREAD

**Discussion by lokeshwarobuli** [mswjs/msw#1559] [Q&A]
I am trying to make a Axios call  and mocking that with the MSW, but I get an error stating that it failed to mock because of some CORS settings

```
import { rest } from 'msw'

export const handlers = [
    rest.get('http://localhost:8000/api/score', (req, res, ctx) => {
        return res(
            ctx.status(200),
            ctx.json({
                id: '920854',
                score: 85,
            }),
        )
    }),
]
```

can you someone help me out with correct settings !!

**Comment by kettanaito** [comment]
Hi, @lokeshwarobuli. Can you please share how you're making the request? I'm particularly interested in the host you're running your app on. I assume it's something different than `localhost:8000`, thus a CORS violation occurs. 

Ideally, if you could provide a reproduction repository, I'd help you figure this out.

**Reply by lokeshwarobuli** [reply]
Yeah I am using `localhost:8010` and probably that's why it's failing, how to handle that scenario ?? Thanks for catching that quickly @kettanaito

**Reply by lokeshwarobuli** [reply]
This was exactly the method of the API call, that I am trying to mock 

```
const BASE_URL: string = `http://localhost:8000`;

interface Score {
    id: string;
    score: number;
}

export async function getScore(): Promise<Score> {
    try {
        const response: Score = await axios.get(`${BASE_URL}/api/score`);
        return response;
    } catch (error) {
        throw error;
    }
}
```

**Reply by lokeshwarobuli** [reply]
I am running the front-end in `localhost: 3005` and backend in `localhost:8010`

**Comment by hlopes-newday** [comment]
I'm interested in an answer to this as well. We're mocking requests to Sanity while locally running msw inside jest tests and also getting a CORS error.

**Reply by kettanaito** [reply]
Please provide a reproduction repository so I could help resolve this. Thanks!
